src/components/DecodeTxs/index.tsx

```diff
@@ -101,25 +101,27 @@ export const BasicTxInfo = ({
 export const getParameterElement = (parameter: DecodedDataBasicParameter, index: number): ReactElement => {
   let valueElement
 
-  if (parameter.type === 'address') {
-    valueElement = (
-      <PrefixedEthHashInfo
-        hash={parameter.value}
-        showAvatar
-        textSize="lg"
-        showCopyBtn
-        explorerUrl={getExplorerInfo(parameter.value)}
-      />
-    )
-  }
-
-  if (parameter.type === 'bytes') {
-    valueElement = (
-      <FlexWrapper margin={5}>
-        <Text size="lg">{getByteLength(parameter.value)} bytes</Text>
-        <CopyToClipboardBtn textToCopy={parameter.value} />
-      </FlexWrapper>
-    )
+  if (!Array.isArray(parameter.value)) {
```

Comment:
I would check whether it is an array and go purely by that instead of whether it ends with `('[]')` (further down) to be safer.

Reply:
As the `parameter.type.endsWith('[]')` check was here before I am afraid it was catching some discrepancy between the param value and type in some cases. However, I am good removing it ....

Reply:
If it's legacy, we can leave it but I feel like checking the value is the safest approach. If we try to render an array, the Safe will crash. You could check alongside it? `parameter.type.endsWith('[]')  || Array.isArray(parameter.value)`

Reply:
@iamacook it's already checking for concrete types "address" and "bytes". Your suggestion is redundant.

Reply:
I'm referencing further down. It [could render an array](https://github.com/gnosis/safe-react/blob/b200aad57fd86ea45ec5d8625313c66bde0249ff/src/components/DecodeTxs/index.tsx#L128-L134) based purely on the label.